### DISCUSSION TITLE
Inference in JAX?

### DISCUSSION THREAD

**Discussion by hayden-donnelly** [jax-ml/jax#17193] [Q&A]

**Comment by jakevdp** [comment]
Hi – I don't think there's enough information here to answer your question. "Inference" is a pretty broad term – the answer to your question would depend on what model/algorithm you're using, what hardware you're running on, and how effectively your model is implemented on JAX and Tensorflow. But I can't think of any reason *a priori* to expect Tensorflow to be faster than JAX, taking into account all those variables.
